symfony/plugins/orangehrmLeavePlugin/Service/LeaveAssignmentService.php

```diff
@@ -19,101 +19,107 @@
 
 namespace OrangeHRM\Leave\Service;
 
+use DateTime;
+use OrangeHRM\Core\Traits\Auth\AuthUserTrait;
+use OrangeHRM\Core\Traits\ORM\EntityManagerHelperTrait;
+use OrangeHRM\Entity\Employee;
+use OrangeHRM\Entity\Leave;
+use OrangeHRM\Entity\LeaveRequest;
+use OrangeHRM\Entity\LeaveRequestComment;
+use OrangeHRM\Entity\LeaveStatus;
+use OrangeHRM\Entity\WorkflowStateMachine;
 use OrangeHRM\Leave\Dto\LeaveParameterObject;
+use OrangeHRM\Leave\Exception\LeaveAllocationServiceException;
+use OrangeHRM\Leave\Traits\Service\LeaveEntitlementServiceTrait;
 
-class LeaveAssignmentService extends AbstractLeaveAllocationService {
+class LeaveAssignmentService extends AbstractLeaveAllocationService
+{
 
     protected $leaveEntitlementService;
```

Comment:
read https://www.php-fig.org/psr/psr-12 `4.2 Using traits` section
> The use keyword used inside the classes to implement traits MUST be declared on the next line after the opening brace.

And remove this property since not using

Reply:
done